server/image_remove_libpod.go

```diff
@@ -0,0 +1,49 @@
+// +build libpod
+
+package server
+
+import (
+	"time"
+
+	"github.com/containers/storage"
+	"github.com/projectatomic/libpod/libpod/image"
+	"github.com/sirupsen/logrus"
+	"golang.org/x/net/context"
+	pb "k8s.io/kubernetes/pkg/kubelet/apis/cri/runtime/v1alpha2"
+)
+
+// RemoveImage removes the image.
+func (s *Server) RemoveImage(ctx context.Context, req *pb.RemoveImageRequest) (resp *pb.RemoveImageResponse, err error) {
+	const operation = "remove_image"
+	defer func() {
+		recordOperation(operation, time.Now())
+		recordError(operation, err)
+	}()
+
+	logrus.Debugf("RemoveImageRequest: %+v", req)
+
+	options := storage.StoreOptions{
+		RunRoot:   s.Config().RunRoot,
+		GraphRoot: s.Config().RootConfig.Root,
+	}
+
+	r, err := image.NewImageRuntimeFromOptions(options)
```

Comment:
Running this on every operation seems inefficient.  I'd rather do this at start-up and stash the image runtime on the `Server` struct or some such.  Running this on every operation seems like a reasonable approach during the early-WIP phase of this PR, but moving to the stash-on-a-struct approach soon would also save you some repetitive code in these `Server` methods.

Reply:
@wking I agree, and I think the Server struct is probably right place too.  However, I was thinking about initially not changing that struct until things were further along in part because it would have to be done conditionally for the libpod builtag.

Given that, what is your preference?  Do it anyways?

Reply:
implemented a getter as we discussed on IRC